### PR TITLE
Feature/better error logs

### PR DESCRIPTION
**Pull request by jgrayauskie**
Seeing some errors logging in to gov env apps.

This will help us narrow in on what is failing.

we are seeing `failed to fetch user profile` in the logs and we dont know which is the issue since they both have the same error staring 😭

### CODE REVIEW

**Review by nikkis-acc — APPROVED** [review]
I've never seen this project before, interesting!! Can't see anything that would break though, all just console logs, so I'm happy with it.

File: lib/strategy.js [hunk 1]
```diff
@@ -171,7 +171,9 @@ AcpStrategy.prototype.authenticate = function(req, options = {}) {
 				null,
 				function (err, body, res) {
 	              if (err) {
-					  return self.error(new InternalOAuthError('failed to fetch user profile', err));
+					  console.log('Request to ', userInfoURL, ', failed ', res.statusCode);
```
Comment (olivetreee) [inline lib/strategy.js]: please safe get from `res` here. or check in oauth2 docs if `res` should always be present

Comment (olivetreee) [reply lib/strategy.js]:
```suggestion
					  console.log('Request to ', userInfoURL, ', failed with status ', res.statusCode);
```

File: lib/strategy.js [hunk 2]
```diff
@@ -188,6 +190,8 @@ AcpStrategy.prototype.authenticate = function(req, options = {}) {
 						null,
 						function (err, redirectBody, res) {
 							if (err || !(res.statusCode >= 200 && res.statusCode <= 299)) {
+								console.log('Redirect request to ', res.headers.location, ', failed ', res.statusCode);
```
Comment (olivetreee) [inline lib/strategy.js]: same safe get comment from above.
also, just noticed the function on L191 receives `res` as the 3rd param, but `res` also exists in the outer scope, like on L181. just wanna make sure we're looking at the proper `res` object

**Comment by olivetreee** [comment]
btw, L88 logs some errors when fetching the accessToken. did you check for those as well? if we can fetch tokens properly, and there's issue is w/ fetching the user-info from IS, I'd be inclined to say it's on the service.

also, the OIC_ISSUER from atlas-config is `https://login.myaccolade.com/` (the userInfoURL ends up being `https://login.myaccolade.com/user-info` if I'm not mistaken). should it be a gov URL instead?

File: lib/strategy.js (hunk 2 above)
Comment (jgrayauskie) [reply lib/strategy.js]: Yeah i will add the saftey, this code was all copied from the original package, which is why i dont think it was there to begin with

**Review by acc-nicholas — APPROVED** [review]

**Pull request merged**